yt/frontends/cf_radial/data_structures.py

```diff
@@ -278,10 +278,10 @@ def _parse_parameter_file(self):
             self.domain_left_edge = np.array([x.min(), y.min(), z.min()])
             self.domain_right_edge = np.array([x.max(), y.max(), z.max()])
             self.dimensionality = 3
-            dims = [xr_ds_handle.dims[d] for d in "xyz"]
+            dims = [xr_ds_handle.sizes[d] for d in "xyz"]
```

Comment:
this is a fix for the deprecation warning coming from xarray -- note that the sizes attribute has been around a long time (xarray vs 0.7ish maybe?) so no worry about backwards compatibility in yt.